### PR TITLE
updated resources

### PR DESCRIPTION
**Pull request by isha614**
## PR Checklist

- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [ ] New features are documented, with examples if plot related.
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] Conforms to Matplotlib style conventions (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

**Review by jklymak — APPROVED** [review]
I checked both links and they are legitimate.  

Were we thinking of moving this page?

File: doc/resources/index.rst [hunk 1]
```diff
@@ -54,6 +54,12 @@ Videos
 * `Data Visualization Basics with Python (O'Reilly)
   <https://www.oreilly.com/library/view/data-visualization-basics/9781771375573/>`_
   by Randal S. Olson
+* `Matplotlib Introduction
+  <https://www.youtube.com/playlist?list=PLeo1K3hjS3uu4Lr8_kro2AqaO6CFYgKOl>`_
+  by codebasics
+* `Mtaplotlib
```
Comment (aitikgupta) [inline doc/resources/index.rst:60]:
```suggestion
* `Matplotlib
```
Didn't know we track these.. but typo anyway

**Comment by tacaswell** [comment]
@meeseeksbot backport to v3.5.x

**Comment by timhoffm** [comment]
@meeseeksdev backport to v3.5.x

**Pull request merged**